src/types/UserResponce.ts

```diff
@@ -0,0 +1,10 @@
+export interface UserResponce {
```

Comment:
Typo in the interface name, it should be 'UserResponse' instead of 'UserResponce'.

Reply:
updated